.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+## Description
+Please include a summary of the change, motivation and context.
+
+<!--
```

Comment:
Unless I'm mistaken, the template can break these things down so that you don't need the user to make a judgement for each section. Ex here's an issue template that already routes context towards bug vs feature https://github.com/openzipkin/zipkin/tree/master/.github/ISSUE_TEMPLATE